src/list.js

```diff
@@ -0,0 +1,102 @@
+let listArr = JSON.parse(localStorage.getItem('task')) || [];
+export default class List {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  addTask = () => {
+    document.querySelector('form').addEventListener('submit', () => {
+      const description = document.getElementById('new-item').value;
+      const completed = false;
+      const index = listArr.length;
+      if (description) {
+        const newTask = {
+          description,
+          completed,
+          index,
+        };
+        listArr.push(newTask);
+        localStorage.setItem('task', JSON.stringify(listArr));
+        this.showTask();
+      }
+    });
+  };
+
+  showTask = () => {
+    if (!listArr.length) {
+      document.querySelector('#item-list').innerHTML = 'No task Added';
+    } else {
+      const doList = document.querySelector('#item-list');
+      let li = '';
+      listArr.forEach((elem, index) => {
+        li += `
+        <li class="todo" id="todo">
+        <button class="tskTog" type="button" title="check!" alt="check!" tabindex="0" id="emi">
+        </button>
+        <button class="tskTog2" type="button" title="check!" alt="check!" tabindex="0" id="emi2">
+          <i class="fa-sharp fa-solid fa-square-check"></i>
+        </button>
+        
+        <div class="view">
+          <label class="label" tabindex="0" for="${elem.index + index}">${elem.description}</label>
```

Comment:
- [ ] Great job so far, after marking a task as completed and refreshing the page the local storage shows it's marked as completed but the checkbox status(checked) doesn't persist, Kindly fix that.

<img width="869" alt="image" src="https://user-images.githubusercontent.com/59416136/221288186-3a4ff387-274e-45c6-b878-3bc1b852b3c2.png">